### PR TITLE
[KOSTALINVERTER] Add product image for 3. generation

### PR DESCRIPTION
**Pull request by RS84**
[KOSTALINVERTER] Add product image for 3. generation

This PR just adds a picture of the device, which was missing in PR #4632 
The picture was provided by KOSTAL (represented by the project manager) right the day after the PR was merged, so it could not be included in the original PR

Picture looks great =)

### CODE REVIEW

File: addons/binding/org.openhab.binding.kostalinverter/README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ Scrapes the web interface of the inverter for the metrics of the supported chann
 
 ![Kostal Pico](doc/kostalpico.jpg)
 
+![Kostal PLENTICORE / PIKI IQ](doc/PLENTICORE.tif)
```
Comment (Hilbrand) [inline addons/binding/org.openhab.binding.kostalinverter/README.md]: Can you make it a jpg. It's now a rather large image. (also maybe use lowercase for filename).

Comment (RS84) [reply addons/binding/org.openhab.binding.kostalinverter/README.md]: done :)

Comment (Hilbrand) [reply addons/binding/org.openhab.binding.kostalinverter/README.md]: Hmm. I would have expected the image to be smaller,  but it's not much smaller than the tif :confused:. So I imported it in GIMP, were it complains about color space and exported it. This results in a ~45kb image. Can you see if you can reduce the size of the image?

Comment (RS84) [reply addons/binding/org.openhab.binding.kostalinverter/README.md]: 14,6 kB now :)

**Pull request merged**